### PR TITLE
[infomanager] add Container.HasParent info bool

### PR DESCRIPTION
**Pull request by mkortstiege**
As requested by @HitcherUK

Forum thread: http://forum.kodi.tv/showthread.php?tid=222058

### CODE REVIEW

File: xbmc/guilib/GUIBaseContainer.cpp [hunk 1]
```diff
@@ -1167,6 +1167,8 @@ bool CGUIBaseContainer::GetCondition(int condition, int data) const
     return (HasNextPage());
   case CONTAINER_HAS_PREVIOUS:
     return (HasPreviousPage());
+  case CONTAINER_HAS_PARENT_ITEM:
+    return (m_items.size() && m_items[0]->IsFileItem() && (std::static_pointer_cast<CFileItem>(m_items[0]))->IsParentFolder());
```
Comment (xhaggi) [inline xbmc/guilib/GUIBaseContainer.cpp]: what happens if you disable the parent item "..." or I'm  wrong here?

Comment (xhaggi) [inline xbmc/guilib/GUIBaseContainer.cpp]: I'm wrong you want to return if the list contains a parent item :)

Comment (mkortstiege) [inline xbmc/guilib/GUIBaseContainer.cpp]: Jep. Sorry for being too lazy to write a proper description.

**Comment by hudokkow** [comment]
jenkins build this please

**Comment by HitcherUK** [comment]
@mkortstiege Confirmed working, thanks.

**Comment by mkortstiege** [comment]
@BigNoid, @ronie objections?

**Comment by ronie** [comment]
not at all.

**Comment by BigNoid** [comment]
No objections from me.

**Comment by mkortstiege** [comment]
Alright. jenkins build and merge

**Pull request merged**